current-spotify-song/current-spotify-song

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/python3
+#
+# Copyright 2020 Andreas Palm <https://github.com/zapling/>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of this software and
+# associated documentation files (the "Software"), to deal in the Software without restriction,
+# including without limitation the rights to use, copy, modify, merge, publish, distribute,
+# sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all copies or
+# substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+# BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+# NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+# DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import os
+import sys
+import json
+import pathlib
+import urllib.parse
+import urllib.request
+import http.server
+
+REDIRECT_URI = 'http://localhost:8080/'
+
+""" HTTP request wrapper """
+def request(url, headers, body=None):
+  if body is not None:
+    body = urllib.parse.urlencode(body).encode()
+
+  req = urllib.request.Request(
+    url=url,
+    data=body,
+    headers=headers
+  )
+
+  try:
+    res = urllib.request.urlopen(req)
+    return {'code': res.status, 'data': json.loads(res.read())}
+
+  except urllib.error.HTTPError as e:
+    return {'code': e.status, 'data': e.reason}
+
+""" Get the path to the configuration folder """
+def getConfigurationFolder():
+  path = str(pathlib.Path.home()) + '/.config/current-spotify-song/'
```

Comment:
The data stored isn't really configuration files, maybe these files should be saved to `~/.local/share/current-spotify-song` instead?